beacon-chain/blockchain/execution_engine.go

```diff
@@ -70,7 +70,6 @@ func (s *Service) notifyForkchoiceUpdate(ctx context.Context, arg *notifyForkcho
 	hasAttr, attr, proposerId, err := s.getPayloadAttribute(ctx, arg.headState, nextSlot)
 	if err != nil {
 		log.WithError(err).Error("Could not get head payload attribute")
-		return nil, nil
```

Comment:
So if there was an error, wouldn't `attr` be nil? What is the expected result of sending the EL a nil forkchoice?

Reply:
Discussed offline and verified with spec. `null` / `nil` is an acceptable parameter for this call.

https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md#engine_forkchoiceupdatedv1